app/javascript/components/UploadButton.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const UploadButton = ({ onChange }) => {
+  return (
+    <div className='row'>
+      <div className="file-upload-div col-12 d-flex justify-content-end align-items-center px-1 mb-2">
+        <div className="file-upload btn btn-sm btn-light-primary m-0">
+          <span>Upload Photo</span>
+          <input type="file" className='upload' onChange={e => onChange(e)} />
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export { UploadButton };
```

Comment:
`export { UploadButton };` is the same as `export UploadButton;`
no need to generate extra objects for exports